pages/community/events/index.js

```diff
@@ -0,0 +1,207 @@
+      description="Our catalogs of events and meetups"
+      image={image}
+      hideBanner={true}
+      wide
+    >
+      <div className="mt-10 sm:mt-28">
+        <div className="w-full sm:w-9/12">
+          <h1 className="countdown-text-gradient font-semibold text-3xl lg:text-7xl leading-tight md:text-4xl">
+            Join an AsyncAPI event from anywhere in the world.
+          </h1>
+          <div className="mt-10">
+            <GoogleCalendarButton href="https://calendar.google.com/calendar/u/3?cid=Y19xOXRzZWlnbG9tZHNqNm5qdWh2YnB0czExY0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t" />
+            <ICSFileButton
+              href="https://calendar.google.com/calendar/ical/c_q9tseiglomdsj6njuhvbpts11c%40group.calendar.google.com/public/basic.ics"
+              className="mt-2 md:mt-0 md:ml-2"
+            />
+          </div>
+        </div>
+        <div className="flex justify-end mt-10">
+          <div className="w-[80%] sm:w-1/3">
+            <Heading
+              level="h2"
+              typeStyle="body-lg"
+              textColor="text-gray-700"
+              className="text-slate-500"
+            >
+              “All events/meetings are live streamed to all AsyncAPI social
+              media accounts. To learn more about meetings setup and automation
+              <TextLink
+                href="https://github.com/asyncapi/community/blob/master/MEETINGS_ORGANIZATION.md"
+                target="_blank"
+              >
+                read our FAQ
+              </TextLink>
+              .”
+            </Heading>
+          </div>
+        </div>
+        <hr className="mt-20 border-dotted border-t-2 border-black " />
+      </div>
+      <div className="mt-24 w-full h-auto sm:h-[400px]">
+        <div className="w-full h-full bg-confBg rounded-md bg-cover bg-center p-10 flex flex-col justify-between text-white">
+          <img
+            src="/img/logos/conflogo.png"
+            alt="conf-logo"
+            className="w-[100px] sm:w-[150px]"
+          />
+          <div className="w-full sm:w-[70%]">
+            <Heading
+              level="h2"
+              typeStyle="heading-sm-semibold"
+              className="mt-10 lg:text-2xl"
+            >
+              Watch the AsyncAPI 2022 conference recordings from anywhere around
+              the world for free
+            </Heading>
+            <a
+              href="https://www.youtube.com/watch?v=NTHsezlKBh8&list=PLbi1gRlP7pijRiA32SU36hD_FW-2qyPhl"
+              target="_blank"
+              rel="noreferrer"
+            >
+              <div className="mt-5 flex items-center">
+                <span>Watch now</span>
+                <ArrowRightIcon className="w-[20px] ml-3 mt-1" />
+              </div>
+            </a>
+          </div>
+        </div>
+      </div>
+      <div className="mt-20">
+        <div className="sm:flex justify-between items-center">
+          <Heading level="h2" typeStyle="heading-md">
+            All Events
```

Comment:
do you have some paging in mind? by default, you show all events, not just upcoming as we have on the current website. This means a bunch of meetings 😄 

also remember that the current [meetings.json](https://github.com/asyncapi/website/blob/master/config/meetings.json) file is designed to hold only meetings from the future. If you also want to have past meetings, you need to do some modifications in https://github.com/asyncapi/website/blob/master/scripts/build-meetings.js#L26

Reply:
I will only be listing events ranging from 4-5 months back, then add an archive section later on (I'd use lazy loading)

Reply:
> also remember that the current [meetings.json](https://github.com/asyncapi/website/blob/master/config/meetings.json) file is designed to hold only meetings from the future. If you also want to have past meetings, you need to do some modifications in https://github.com/asyncapi/website/blob/master/scripts/build-meetings.js#L26

I already did the modification